{"uuid": "214d4382-1553-4be2-bf87-ad4d47f52913", "children": ["e9d29f5b-5dd5-4988-965b-964c47f28124"], "befores": [{"name": "create_driver", "status": "passed", "start": 1737707701991, "stop": 1737707721759}], "afters": [{"name": "create_driver::0", "status": "broken", "statusDetails": {"message": "AttributeError: 'Function' object has no attribute 'rep_call'\n", "trace": "  File \"/Users/mohsenakbari/Downloads/Python-Selenium-Pytest-main/.venv/lib/python3.10/site-packages/allure_commons/_allure.py\", line 185, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/Users/mohsenakbari/Downloads/Python-Selenium-Pytest-main/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 911, in _teardown_yield_fixture\n    next(it)\n  File \"/Users/mohsenakbari/Downloads/Python-Selenium-Pytest-main/tests/conftest.py\", line 72, in create_driver\n    if request.node.rep_call.failed:\n"}, "start": 1737707721780, "stop": 1737707721781}], "start": 1737707701991, "stop": 1737707721781}